gulpfile.js

```diff
@@ -180,11 +177,18 @@ gulp.task('serve.https', function() {
     }));
 });
 
+gulp.task('node-test', gulp.series(build, function() {
+  return gulp.src('./test/endpoint.js')
+    .pipe(mocha({reporter: 'spec'}));
+}));
+
+exports.build = build
+
 // Macro-tasks
-gulp.task('bookmarklets', ['bookmarklet.dev', 'bookmarklet.prod', 'bookmarklet.edge']);
-gulp.task('test', function(next) {
-  seq('browser-test', 'node-test', next);
-});
-gulp.task('work', ['watch', 'serve']);
-gulp.task('https', ['watch', 'serve.https']);
-gulp.task('default', ['lint', 'test', 'build']);
+exports.bookmarklets = gulp.series('bookmarklet.dev', 'bookmarklet.prod', 'bookmarklet.edge');
+test = gulp.series('browser-test', 'node-test');
```

Comment:
Why this global variable?

Reply:
do you mean 'test'?

Reply:
Yes, why is `test` a global variable here?

Reply:
Its used just below in exports.default

Reply:
Yes, I get it. What I don't get is why you make it a global variable. Haven't you forgotten a `var` here or is this intended?

Reply:
test is used a few lines below for exports.default

Reply:
It should be `var test = gulp.series('browser-test', 'node-test');` nonetheless. The same can be said for `build` line 125 :)

Reply:
Sorry, commenting went a bit odd! Got you, should be fixed now.